.github/workflows/meson.yml

```diff
@@ -112,7 +115,7 @@ jobs:
           esac
 
       - name: Set up WrapDB
-        if: matrix.config.pkgmanager == 'choco' || matrix.config.os == 'macos-latest'
+        if: matrix.config.pkgmanager == 'choco' || matrix.config.os == 'macos-latest' || matrix.config.msvc
```

Comment:
The `matrix.config.pkgmanager == 'choco'` part is actually a leftover from before the MSys dep reuse and nothing sets this now. Either MSVC jobs should now again declare their pkgmanager to be `choco` and there’s no need to add `|| matrix.config.msvc`, or the `choco` condition removed here